samples/jenkins/README.md

```diff
@@ -1,5 +1,370 @@
+then, using `archiveArtifacts` step, attach the archive to the Jenkins build as a build artifacts.
+5. Cleanup - executes [cleanup.sh](scripts/dev/cleanup.sh) to delete all models, assets and jobs from the project.
+
+### Test pipeline
+Import of the artifacts from the previous job, model deployment and evaluation is defined in the test pipeline with the following stages:
+1. Install cdpctl - same as in the development pipeline 
+2. Import assets - using `copyArtifacts` step downloads artifacts archived in the development pipeline, then imports the archive to 
+the newly created deployment space with [create_test_space.sh](scripts/test/create_test_space.sh) script.
+3. Deploy model - deploys model with [create_model_batch_deployment.sh](scripts/test/create_model_batch_deployment.sh) script.
+4. Evaluate model - evaluates model deployment using [create_script_batch_deployment.sh](scripts/test/create_script_batch_deployment.sh) script.
+5. Cleanup - executes [cleanup.sh](scripts/test/cleanup.sh) to delete whole space with all assets.
+
+### Production pipeline
+Same as previously, the preparation part defines the default execution node and loads the secret file. Remaining steps: 
+1. Install cpdctl - same as in the development pipeline
+2. Import assets - update existing AI assets by importing them from the archive stored in the artifacts repository using 
+[import_to_prod_space.sh](scripts/prod/import_to_prod_space.sh) script.
+3. Update model deployment - executes [update_model_deployment.sh](scripts/prod/update_model_deployment.sh) script to redeploy the new improved model.
+4. Evaluate the model - run regression tests to ensure the quality of the new model version using [run_model_deployment_validation.sh](scripts/prod/run_model_deployment_validation.sh) script, then attach the evaluation
+results to the Jenkins job with `archiveArtifacts` step.
+5. Update evaluation script deployment - redeploy the evaluation script using [update_script_deployment.sh](scripts/prod/update_script_deployment.sh) script.
+
+## Run the scenario
+When all configuration steps are done, it is a time to execute the scenario and look at the results.
+
+### Development pipeline run
+[Run](https://www.jenkins.io/doc/book/pipeline/getting-started/#through-the-classic-ui) development pipeline job with default parameters
+and wait for completion:
+
+  ![dev-pipeline.png](img/dev-pipeline.png)
+
+Artifacts created by the specific run can be found in the build details:
+
+  ![development-build.png](img/development-build.png)
+
+In the logs from the execution, available in the job `Console Output`, there are detailed outputs from bash scripts used 
+by the pipeline, the most important logs are:
+- CPDCTL installation and version
+```text
+bash -e ./samples/jenkins/scripts/helpers/install_cpdctl.sh
+cpdctl
+Installed cpdctl in version:
+1.15.0
```

Comment:
Currently most recent version is 1.1.15. It's not a big deal but a hint that the output is faked.